readme.md

```diff
@@ -385,31 +432,34 @@ test(async t => {
 });
 ```
 
-*You don't have to manually call `t.end()`.*
-
 ### Observable support
 
 AVA comes with builtin support for [observables](https://github.com/zenparsing/es-observable).
+If you return an observable from a test, AVA will automatically consume it to completion before ending the test.
+*You don't have to use legacy `test.async` mode or `t.end()`.*
 
 ```js
 test(t => {
-	return Observable.of(1, 2, 3).map(n => {
-		t.true(n > 0);
-		return n * n;
-	});
+	t.plan(3);
+	return Observable.of(1, 2, 3, 4, 5, 6)
+		.filter(n => {
+			// only even numbers
+			return n % 2 === 0;
+		})
+		.map(n => {
+			t.pass();
```

Comment:
You could change map with `.do(() => t.pass());`. Otherwise, looks good.

RxJS has some [docs](https://github.com/ReactiveX/RxJS/blob/master/doc/writing-marble-tests.md) on marble test. But I'm not sure how they would fit with AvA